tests/integration_tests/security/test_jail.py

```diff
@@ -81,6 +66,7 @@ def test_empty_jailer_id(test_microvm_with_api):
     test_microvm.jailer = JailerContext(
         jailer_id="",
         exec_file=fc_binary,
+        netns=None,
```

Comment:
Here is a `netns=None`

Reply:
Yep, I think that could be dropped, is not important for the test.

Reply:
fixed